ff/serving/python_client/sdk/transformations.py

```diff
@@ -0,0 +1,49 @@
+transformations_to_register = []
+
+class Transformations:
+    def __init__(self, feature: bool, name: str, variant = str, inputs = list, 
+                        entity = str, fn_type = str, executor = str, frequency = str, ft_type = str):
+        self.feature = feature
+        self.name = name
+        self.variant = variant
+        self.inputs = inputs
+        self.entity = entity
+        self.fn_type = fn_type
+        self.frequency = frequency
+        self.ft_type = ft_type
+
+def transformation(*args, **kwargs):
+
+    def add_transformation(definition_func):
+
+        transformation = Transformations(kwargs['feature'], definition_func.__name__ ,
```

Comment:
Could we figure out how to actually save the python code from the function, in this case:
    count = 10000000000
    return count